src/networks/devnet/mod.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
```

Comment:
Some of the content from the meeting earlier this week belongs here:
- what is a devnet?
- how is it expected to be used?
- design choices:
  - named devnets vs infinitely configurable ones

...etc

Reply:
https://github.com/ChainSafe/forest/issues/3054

I'd rather this not be in the source code but in the developer handbook, but it's something secondary.